apps/guardian-ui/src/components/DangerZone.tsx

```diff
@@ -1,16 +1,18 @@
 import React from 'react';
 import { Box, Text, Flex, theme } from '@chakra-ui/react';
-import { ClientConfig } from '@fedimint/types';
 import { GuardianAuthenticationCode } from './GuardianAuthenticationCode';
 import { DownloadBackup } from './DownloadBackup';
 import { useTranslation } from '@fedimint/utils';
 
 interface DangerZoneProps {
-  config: ClientConfig | undefined;
   ourPeer: { id: number; name: string } | undefined;
+  inviteCode: string;
 }
 
-export const DangerZone: React.FC<DangerZoneProps> = ({ config, ourPeer }) => {
+export const DangerZone: React.FC<DangerZoneProps> = ({
```

Comment:
nit: Would it make sense to collapse these options by default? 
Just thinking the buttons should be hidden from the main dashboard